net/scripts/gce-provider.sh

```diff
@@ -171,18 +173,23 @@ cloud_CreateInstances() {
   declare optionalAdditionalDiskSize="${11}"
   declare optionalPreemptible="${12}"
   #declare sshPrivateKey="${13}"  # unused
+  declare optionalImageName="${14}"
 
-  if $enableGpu; then
-    # Custom Ubuntu 20.04 LTS image with CUDA 10.2 installed
-    #
-    # Unfortunately this image is not public.  When this becomes an issue, use
-    # the stock Ubuntu 20.04 image and programmatically install CUDA after the
-    # instance boots
-    #
-    imageName="ubuntu-2004-focal-v20201211-with-cuda-10-2 --image-project principal-lane-200702"
+  if [[ -n $optionalImageName ]]; then
+    imageName=$optionalImageName
```

Comment:
If I remember correctly ubuntu images have different set of packages pre-installed, which make the scripts fail if you just change the image name.

e.g. if you run this scripts with `ubuntu-2110-impish-v20220309` it will fail because impish image does not have `bzip2` or `at` pre-installed.

So the scripts don't simply support an arbitrary ubuntu image name, which might make this argument confusing or less useful; so rather than adding this option, might be better just to upgrade the ubuntu image to `ubuntu-2004-focal-v20220419`

Reply:
I would be happy to upgrade than